docs/csharp/getting-started/index.md

```diff
@@ -5,18 +5,37 @@ description: Find short, simple tutorials that let you quickly learn introductor
 keywords: C#, Get Started, Acquisition, Install
 author: rpetrusha
 ms.author: ronpet
-ms.date: 05/02/2017
+ms.date: 08/23/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: devlang-csharp
 ms.devlang: csharp
+translation.priority.ht: 
```

Comment:
don't add this, let's follow up with loc on this metadata

But you need to bring the helpviewer_keywords from the other topics. They are for the offline book index.